src/NServiceBus.Persistence.CosmosDB/Outbox/OutboxPersister.cs

```diff
@@ -90,7 +90,7 @@ public async Task SetAsDispatched(string messageId, ContextBag context)
         readonly JsonSerializer serializer;
         readonly int ttlInSeconds;
 
-        internal static readonly string SchemaVersion = "1.0.0";
+        internal static readonly string SchemaVersion = "0.1.0";
```

Comment:
As an alternative we could also just call it preview or 1.0.0-preview. I don't think this will change much at all between preview versions because the current schema is super lightweight

Reply:
How about adding as well a comment with something like

```
// The version here represents the schema version and not necessarily the assembly version!
```